src/io/jesi/backpack/miscellaneous.cljc

```diff
@@ -70,3 +71,12 @@
     :default sym))
 
 ;TODO create a macro that take a form and can transform it to the required runtime e.g. cljs ns rename, cljs catch clause
+
+(defn collify
+  "Puts value `v` in a vector if it is not a collection. Returns `nil` if no value"
+  ([] nil)
+  ([v]
+   (condp call v
+     nil? nil
```

Comment:
Not 100% convinced that this should return `nil`, but `(= nil (seq []))`. so everything that takes a collection _should_ handle `nil`

Reply:
Give nil get nil